classes/form/cms_types_form.php

```diff
@@ -52,6 +54,27 @@ protected function definition() {
         $mform->addElement('editor', 'description', get_string('description'));
         $mform->setType('description', PARAM_RAW);
 
+        $mform->addElement(
+            'textarea',
+            'mustache',
+            get_string('mustache', 'cms'),
+            ['rows' => 10]
+        );
+        $mform->setType('mustache', PARAM_RAW);
+
+        // Generate the help text for mustache template.
+        $cmstype = new cms_types();
+        $cms = $cmstype->get_sample_cms();
```

Comment:
can you please convert this into the format that it gets used in mustache, so instead of

```
{
    "site": {
        "fullname": "Evergreen",
        "shortname": "Evergreen",
        "wwwroot": "https:\/\/master.localhost"
    }
}
```
you have a html table like this showing the properly non escaped way it will really appear:
```
{{site.fullname}}   Evergreen
{{site.shortname}}  Evergreen
{{site.wwwroot}}    https://master.localhost
```

Reply:
done